src/components/Equations.ts

```diff
@@ -98,27 +99,27 @@ export class Equations<TGameStartr extends FullScreenPokemon> extends Component<
      * @param level   The level of the new Pokemon (by default, 1).
      * @param moves   What moves the Pokemon has (by default, generated from its type
      *                and level).
-     * @param iv   What IV points the Pokemon should start with (by default, generated 
+     * @param iv   What IV points the Pokemon should start with (by default, generated
      *             from the newPokemonIVs equation.
      * @param ev   What EV points the Pokemon should start with (by default, generated
      *             from the newPokemonEVs equation).
      * @returns A newly created Pokemon.
      */
-    public newPokemon(title: string[], level?: number, moves?: IMove[]): IPokemon {
+    public newPokemon(chosenInfo: INewPokemon): IPokemon {
         const ev: IValuePoints = this.newPokemonEVs();
         const iv: IValuePoints = this.newPokemonIVs();
-        level = level || 1;
 
         return {
-            experience: this.experienceStarting(title, level || 1),
+            experience: this.experienceStarting(chosenInfo.title, chosenInfo.level || 1),
             ev,
+            item: chosenInfo.item || undefined,
```

Comment:
No need to `|| undefined`. If it's already type `string[] | undefined` that does nothing.